ssl/d1_lib.c

```diff
@@ -338,14 +342,12 @@ static void dtls1_double_timeout(SSL_CONNECTION *s)
 
 void dtls1_stop_timer(SSL_CONNECTION *s)
 {
-    struct timeval tv;
-
     /* Reset everything */
     s->d1->timeout_num_alerts = 0;
     s->d1->next_timeout = ossl_time_zero();
     s->d1->timeout_duration_us = 1000000;
-    tv = ossl_time_to_timeval(s->d1->next_timeout);
-    BIO_ctrl(s->rbio, BIO_CTRL_DGRAM_SET_NEXT_TIMEOUT, 0, &tv);
+    /* set s->d1->next_timeout into bio interface */
+    dtls1_bio_set_next_timeout(s->rbio, s->d1);
```

Comment:
The comment here (and a similar one elsewhere) probably not necessary as the function name is quite descriptive.